### DISCUSSION TITLE
how do i  use the file in discord

### DISCUSSION THREAD

**Discussion by AaronShajo** [Celienn/Blox-Fruit-Notifier#3] [Q&A]
im new idk how to apply to discord

**Comment by Celienn** [comment]
the bot does not work for the current version of blox fruit i will update it soon
sry :)

**Comment by AaronShajo** [comment]
OK but I don't know how to open one of those files

**Reply by Celienn** [reply]
if you ask how to run the bot even if it don't work 
you need to create a discord app and then put the token in config.json 
==> https://discordpy.readthedocs.io/en/stable/discord.html
then run bot.js with node 
==> https://nodejs.org/en/
